README.md

```diff
@@ -192,62 +192,104 @@ populate the following:
 - `drivers/ojdbc8.jar` - This is the JDBC driver for our Oracle backend and
   can be found for free on [Oracle's site](https://www.oracle.com/ca-en/database/technologies/appdev/jdbc-downloads.html).
 - `secrets/` - These are credential files needed to connect to other systems.
-- `.env` - This file sets a few non-critical but still private options.
+
+## Using the development environment
+
+The Docker Compose stack we use to run Spark (via `docker compose up -d`)
+has a separate development environment that can be used to test code changes
+without disrupting the production containers. The development environment is
+configured via a dedicated
+[env-file](https://docs.docker.com/compose/how-tos/environment-variables/variable-interpolation/#additional-information-1).
+
+To start the development environment, run:
+
+```bash
+docker compose up -d
+```
+
+To submit a job to the development environment, change the container target
+of your command from `prod` to `dev`. For example:
+
+```bash
+docker exec -it spark-node-master-dev ./submit.sh \
+    --json-string "$(yq -o=json .test_jobs ./config/default_jobs.yaml)"
+```
+
+A typical development workflow might look something like:
+
+1. Clone the repository to your own machine or home directory. Do _not_ use
+  the production `shiny-server` copy of the repository for development.
+2. Copy the secrets and drivers from the production setup to the development
+  repository. See [Files not included](#files-not-included).
```

Comment:
Added in c93facc.

Reply:
You might have caught this already, but I think we want `cp` instead of `mv` here: https://github.com/ccao-data/service-spark-iasworld/blob/c93faccc221d605081b9ac86efebd497dba88a42/README.md?plain=1#L224-L225